src/murfey/cli/lif_to_tiff.py

```diff
@@ -1,14 +1,18 @@
 import argparse
 from pathlib import Path
 
-from murfey.util.lif import convert_lif_to_tiff
+from murfey.util.clem import convert_lif_to_tiff
 
 
 def run():
-    parser = argparse.ArgumentParser(description="Convert LIF to TIFF")
+    parser = argparse.ArgumentParser(
+        description="Convert LIF files into TIFF image stacks"
+    )
 
     parser.add_argument(
-        nargs=1, dest="lif_path", help="Path to LIF file for conversion"
+        nargs=1,  # nargs=1 creates a 1-item list
```

Comment:
Does this work if we just drop the `nargs` option? i.e. does it default to accepting a single argument?